packages/tezos-proxy-server-config/tezos-proxy-server-config.13.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.9" }
+  "tezos-base" { = version }
+  "tezos-stdlib-unix" { = version }
+  "tezos-test-helpers" { with-test & = version }
```

Comment:
```suggestion
  "uri" {>= "2.2.0"}
  "tezos-test-helpers" { with-test & = version }
```
Since you use `uri` I suggest to add the dependence explicitly (and maybe also link it explicitly in the corresponding dune file)